src/Resources/views/Common/_addToWishlist.html.twig

```diff
@@ -12,7 +12,7 @@
     {% else %}
         <div class="ui dropdown button bb-wishlist-button">
             <span><i class="icon heart"></i></span>
-            <span class="text">Add to wishlist</span>
+            <span class="text">Add to wishlist test</span>
```

Comment:
Is a word `test` needed here, and also I think you should add this text to a translation file (or make a new task for it).